examples/all-clusters-app/esp32/main/DeviceWithDisplay.cpp

```diff
@@ -180,6 +180,25 @@ class EditAttributeListModel : public TouchesMatterStackModel
                 // update the current humidity here for hardcoded endpoint 1
                 ESP_LOGI(TAG, "Humidity changed to : %d", n);
                 app::Clusters::RelativeHumidityMeasurement::Attributes::MeasuredValue::Set(1, static_cast<int16_t>(n * 100));
+            } else if (name == "SystemMode")
```

Comment:
The branch for "SystemMode" is already there. Maybe put this code in already existing if branch: https://github.com/project-chip/connectedhomeip/blob/master/examples/all-clusters-app/esp32/main/DeviceWithDisplay.cpp#L194 The same thing for "OccupiedCoolingSetpoint" and "OccupiedHeatingSetpoint".

Reply:
Actually I am on an older sha so it doesn't have the changes. I will upload the part for system mode